app/storage/dynamo_api.py

```diff
@@ -0,0 +1,71 @@
+from botocore.exceptions import ClientError
+from flask import current_app
+from structlog import get_logger
+
+logger = get_logger()
+
+
+def put_item(table_name, item, overwrite=True):
+    """Insert an item into table"""
+    table = _get_table(table_name)
+
+    put_kwargs = {'Item': item}
+    if not overwrite:
+        first_key = next(iter(item.keys()))
```

Comment:
Can we guarantee that the 1st key is and always will be the table key?

Reply:
I went back and forth on this one.  We can't, but all we need here is a field that is guaranteed to be in all items.  I'll rejig it to explicitly use the key.